docs/developer-docs/latest/concepts/draft-and-publish.md

```diff
@@ -10,15 +10,15 @@ The draft and publish feature allows you to save your content as a draft, to pub
 
 ## Activating or deactivating the draft and publish feature
 
-By default, the draft and publish feature is activated for all newly created collection and single types. It is however possible to deactive the feature at content type level (i.e. it can be activated for one content type, but deactivated for another).
+By default, the draft and publish feature is activated for all newly created collection and single types. It is however possible to deactivate the feature at content type level (i.e. it can be activated for one content type, but deactivated for another).
```

Comment:
```suggestion
By default, the draft and publish feature is enabled for all newly created collection and single types. It is however possible to disable the feature at the content-type level (i.e. it can be enabled for one content type, but disabled for another).
```